cypress/integration/tx_modal.spec.js

```diff
@@ -0,0 +1,192 @@
+const TEST_SAFE = 'rin:0x11Df0fa87b30080d59eba632570f620e37f2a8f7'
+const RECIPIENT_ENS = 'diogo.eth'
+const RECIPIENT_ADDRESS = '0x6a5602335a878ADDCa4BF63a050E34946B56B5bC'
+const SAFE_NONCE = '5'
+
+describe('Tx Modal', () => {
+  before(() => {
+    cy.connectE2EWallet()
+
+    // Open the Safe used for testing
+    cy.visit(`/${TEST_SAFE}`)
+    cy.contains('a', 'Accept selection').click()
+  })
+
+  describe('Send funds modal', () => {
+    it('should open the modal when clicking New Transaction', () => {
+      // Modal should be closed
+      cy.get('[aria-describedby="Send Tokens Form"]').should('not.exist')
+
+      // Opens Tx Modal
+      cy.contains('New Transaction').click()
+      cy.get('[aria-describedby="Send Tokens Form"]').should('be.visible')
+      cy.get('.smaller-modal-window').contains('Send')
+    })
+
+    it('should close the modal by clicking outside of the modal', () => {
+      // Modal is open
+      cy.get('.smaller-modal-window').contains('Send')
+
+      // Click outside of the .smaller-modal-window to close the modal
+      cy.get('.smaller-modal-window').then(($target) => {
+        let coords = $target[0].getBoundingClientRect()
+        const { x, y } = coords
+        cy.get('body').click(x - 10, y - 10)
+      })
+
+      // Modal is closed
+      cy.get('[aria-describedby="Send Tokens Form"]').should('not.exist')
+    })
+
+    it('should close the modal by clicking cancel', () => {
+      // Opens Send Funds modal
+      cy.contains('New Transaction').click()
+      cy.contains('Send funds').click()
+
+      // Send Funds modal is open
+      cy.contains('Send funds').should('be.visible')
+      cy.contains('Step 1 of 2').should('be.visible')
+
+      // Close modal when clicking Cancel
+      cy.contains('Cancel').click()
+
+      // Modal is closed
+      cy.get('[aria-describedby="Send Tokens Form"]').should('not.exist')
+    })
```

Comment:
Like with the previous comment, this isn't worth testing IMHO. The likelihood that the Cancel button breaks is very low. The most important part is that the form, validation, gas and the transaction itself work.

Reply:
Fair enough 👍